api/insight/apiroutes.go

```diff
@@ -283,7 +283,7 @@ func (c *insightApiContext) broadcastTransactionRaw(w http.ResponseWriter, r *ht
 
 	// Respond with hash of broadcasted transaction
 	txidJSON := struct {
-		TxidHash string `json:"txid"`
+		TxidHash string `json:"rawtx"`
```

Comment:
This was just changed in https://github.com/decred/dcrdata/pull/681.  Please do not reintroduce the bug.  Perhaps rebase on master?